### PR TITLE
Updates To Lore

### PR DESCRIPTION
**Pull request by danholden4006**
Updated some Lore stories with correct spelling and grammar. Also added myself to the Credits under Proofreader.

### CODE REVIEW

File: Credits.md [hunk 1]
```diff
@@ -1,199 +1,198 @@
...
-
-## Translators
-[alfaice](https://github.com/alfaice)  
-[AlphaC58](https://github.com/AlphaC58)  
-[ArbiwReist](https://github.com/Meawresion)  
-[CynexXT](https://github.com/CynexXT)  
-[Destro9000](https://github.com/Destro9000)  
-[Divadriel](https://github.com/Divadriel)  
-[EV_Jack](https://github.com/EvJack)  
-[Eziyo](https://github.com/eziyoo)
```
Comment (DexterHuang) [inline Credits.md:32]: Why did you remove this?@@

File: Credits.md [hunk 2]
```diff
@@ -1,199 +1,198 @@
...
-[Seo-na](https://github.com/Seo-na)  
-[siruelionn](https://github.com/siruelionn)  
-SKIFFI  
-[SnRetro](https://github.com/SnRetro)  
-[TheBrainReal](https://github.com/lucasknook)     
-[un-amico](https://github.com/un-amico)  
-Unfairissimo  
-[VerySmashing](https://github.com/VerySmashing)  
-Wanna  
-[Neshaiy](https://github.com/Neshaiy)
```
Comment (DexterHuang) [inline Credits.md:61]: Why did you remove this?@@

**Comment by danholden4006** [comment]
On my end, looking at my files, I did not remove them. All I did was add
myself to the proofreading section. I do not know where or why those were
removed. I apologize greatly if it caused any problems.

**Comment by DexterHuang** [comment]
hmm seems abit weird, do you mind making another PR? I can't see what was changed in this one 0.0
